test/unit/annotation_layer_spec.js

```diff
@@ -23,9 +23,7 @@ describe('Annotation layer', function() {
 
     it('should set and get a valid style', function() {
       var borderStyle = new AnnotationBorderStyle();
-      var dict = new Dict();
-      dict.name = 'D';
-      borderStyle.setStyle(dict);
+      borderStyle.setStyle({ name: 'D' });
```

Comment:
Isn't a `Name` actually what you want to use here, since that is what `dict.get('S')` (in `annotation.js`) will return?
If so, using `borderStyle.setStyle(Name.get('D'));` here should be slightly more clear.